### ISSUE TITLE
Sagui presets should be opt-in not opt-out!

### ISSUE THREAD

**Issue by pirelenito** [saguijs/sagui#109]
I think this is very important to make the project easier to understand and to tweak!

This would change the `webpack.config.js` to:

```js
var sagui = require('sagui')

module.exports = sagui().webpack({
  sagui: {
    /**
     * List of Sagui presets
     */
    enabledPresets: [
      'archetype-library',
      'archetype-pages',
      'babel',
      'base',
      'define-node-env',
      'eslint',
      'images',
      'fonts',
      'json',
      'style',
      'videos'
    ]
  }
})
```

As part of this issue, we might decide to better name or split some of the presets to make it easier to understand what they do.

This also makes it safer to do minor upgrades on any project! As you shouldn't be getting any extra behavior out of the box, unless you explicitly opt-in!

**Comment by pirelenito** [comment]
We should also refactor the `karma` configuration into presets, to allow the same level of granularity configuration.

**Comment by pirelenito** [comment]
Fixed in #110
